### PR TITLE
Add section Environment installation in README.md

### PR DESCRIPTION
**Pull request by alexbeatnik**
Add section Environment installation and Vagrant installation
replase to Environment installation
In section Vagrant update description after runing
Add liks to Vagrant site and Virtualbox site

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,42 @@ Before running any command, please install required Python's dependencies with
 pip install -r requirements.txt
 ```
 
+### Environment installation
```
Comment (extsoft) [inline README.md]:
```suggestion
### System unde test (SUT) installation
```

Comment (extsoft) [inline README.md]: @alexbeatnik Also, please describe which options are available including the case when a user already has a VM with installed docker.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,42 @@ Before running any command, please install required Python's dependencies with
...
+#### Vagrant
```
Comment (extsoft) [inline README.md]: @alexbeatnik Please describe what this type of installation does and, only after, how to get it working.

**Comment by alexbeatnik** [comment]
@extsoft I have no button 'merge'. Can you merge please?

**Comment by extsoft** [comment]
> @extsoft I have no button 'merge'. Can you merge please?

Will do

**Comment by extsoft** [comment]
@alexbeatnik done

**Pull request merged**